### ISSUE TITLE
Can't login to GitHub anymore via Lepton

### ISSUE THREAD

**Issue by UncleMiF** [hackjutsu/Lepton#518]
The more we know about your system and use case, the more easily and likely we can help.

#### Environment info
* OS: macOS 11.5.2
* App Version (`Command/Ctrl + ,`): 1.9.1 (1.9.1)

#### Description of the problem / feature request / question:

Program worked about month ago.
But some days ago it asked me to re-login.
Since the moment, login screen appears, but not disappears after entering login/password.
At the same time the GitHub's Web UI works just fine via browser (I can login there).

#### If possible, provide the log files
For Lepton after v1.9.0, the path of the log file can be found at `Command/Ctrl + ,`
- Mac: `~/Library/Application Support/Lepton/logs`
- Windows: `C:\Users\<username>\AppData\Roaming\Lepton\logs`
- Linux: `~/.cached/Lepton/logs`
>The [debug mode](https://github.com/hackjutsu/Lepton/wiki) is recommended when producing the logs. 
>You can send your log file to cosmo.lepton@gmail.com if don't want to post it here.

2021-09-16T12:18:47.786Z - debug: [REST] [V2] Something wrong happens TypeError: Cannot read property 'body' of undefined. Falling back to [V1]...
2021-09-16T12:18:47.787Z - debug: [REST] [V1] Getting all gists of UncleMiF with token gho_…
2021-09-16T12:18:48.029Z - debug: The gist number on this page is undefined
2021-09-16T12:18:48.287Z - debug: The gist number on this page is undefined
2021-09-16T12:18:48.531Z - debug: The gist number on this page is undefined
2021-09-16T12:18:48.771Z - debug: The gist number on this page is undefined
2021-09-16T12:18:49.021Z - debug: The gist number on this page is undefined
2021-09-16T12:18:49.275Z - debug: The gist number on this page is undefined
(many times)
2021-09-16T12:19:12.427Z - error: The request has failed: TypeError: Cannot convert undefined or null to object
2021-09-16T12:19:12.428Z - debug: -----> Failure with {}
2021-09-16T12:19:12.429Z - error: The request has failed:
{}
2021-09-16T12:19:12.429Z - info: [Dispatch] updateUserSession INACTIVE

**Comment by tellkonkle** [comment]
This likely won't help anyone, but juusssttt in case I'm going to leave this here as a comment. I ran into this problem a few days ago where I couldn't login anymore to GitHub anymore via Lepton after just using Lepton a few days prior. I tried deleting all of these at the same time and it didn't help:

* `C:\Users\<username>\.leptonrc`
* `C:\Users\<username>\AppData\Roaming\Lepton\*`
* My entire Lepton install
* Revoking the OAuth for Lepton

Well, it turns out, I had TWO different Lepton installs running, and the one I was trying to open was using a deprecated OAuth implementation. I can only surmise the following must have happened:

Perhaps I did a manual install from a ZIP (my usual preferred method), and then installed it via an executable later on if I was having an issue with getting the updates to work (I recall a year ago having an issue with the Lepton updates not installing). Or maybe I forgot I used the ZIP installation and then did a regular install into Program Files. Either way I had two installs going. I finally realized this when I noticed the Lepton launcher pinned from my Windows 10 Start Menu was using my preferred dark Lepton theme, while the launcher in my Elgato Stream Deck was using the white theme. It turns out they were pointing to two different executables. 

This is likely a fringe case, but I figured I'd share it in case it has even the remotest chance of helping somebody.
